docs/source/de/add_new_pipeline.md

```diff
@@ -15,7 +15,7 @@ rendered properly in your Markdown viewer.
 
 # Wie erstellt man eine benutzerdefinierte Pipeline?
 
-In dieser Anleitung sehen wir uns an, wie Sie eine benutzerdefinierte Pipeline erstellen und sie auf dem [Hub](hf.co/models) freigeben oder sie der
+In dieser Anleitung sehen wir uns an, wie Sie eine benutzerdefinierte Pipeline erstellen und sie auf dem [Hub](https://huggingface.co/models) freigeben oder sie der
```

Comment:
Good catch, the error was due to `https://` being omitted! Let's keep the `hf.co/models` link since it is shorter and easier to read :)

```suggestion
In dieser Anleitung sehen wir uns an, wie Sie eine benutzerdefinierte Pipeline erstellen und sie auf dem [Hub](https://hf.co/models) freigeben oder sie der
```

Reply:
I was unsure whether to use `hf.co` or `huggingface.co`, so I went with the more commonly used `huggingface.co.` Anyway I have made the modifications as you suggested and updated it accordingly!